src/include/txn.i

```diff
@@ -752,9 +752,15 @@ __wt_txn_read_upd_list(WT_SESSION_IMPL *session, WT_UPDATE *upd, WT_UPDATE **upd
             continue;
         upd_visible = __wt_txn_upd_visible_type(session, upd);
         if (upd_visible == WT_VISIBLE_TRUE) {
-            /* Don't consider tombstone updates for the history store during rollback to stable. */
+            /*
```

Comment:
could you reword the comment, it is not clear.

Reply:
Commented is updated as per the discussion. Thanks.